examples/cpp/inference/infer_multiple_networks.cc

```diff
@@ -164,6 +164,11 @@ int main(int argc, char** argv) {
     return -1;
   }
 
+// Register cpu backend for static linking
+#if defined(NGRAPH_BRIDGE_STATIC_LIB_ENABLE)
+  ngraph_register_cpu_backend();
+#endif
+
```

Comment:
Right now we create only CPU backend here, hence the ifdef is only for CPU. But suggestions for future is we can query the backend we are trying to set/create and register just that backend if NGRAPH_BRIDGE_STATIC_LIB_ENABLE is enabled